tests/test_all.py

```diff
@@ -89,6 +89,29 @@ def test_os_release(auto_container):
         )
 
 
+@pytest.mark.skipif(
+    OS_VERSION in ("15.3", "15.4", "15.5", "basalt"),
+    reason="branding packages are known to not be installed",
+)
+@pytest.mark.parametrize(
+    "container",
+    CONTAINERS_WITH_ZYPPER,
+    indirect=True,
+)
+def test_branding(container):
+    """
+    check that the :file:`/etc/SUSE-brand` file exists and containers SLE branding
+    """
+    location = "/etc/SUSE-brand"
+    branding = "SLE"
+    if OS_VERSION == "tumbleweed":
+        branding = "openSUSE"
+    if OS_VERSION in ("basalt", "tumbleweed"):
+        location = "/usr/etc/SUSE-brand"
```

Comment:
Why use these switches when the tests only run on Tumbleweed?

Reply:
it also runs on 15.6 (and it did until recently basalt completely exploded) also on basalt. eventually I hope to be able to fix it also for older versions, so the `skipif` will reduce over time (famous last words)